spec/features/user_resets_password_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature 'User Resets Password' do
+  scenario' user successfully resets password' do
```

Comment:
Put space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.